### PR TITLE
Bulk Load and cal CSV corrections to address Travis test failure of PR#793

### PR DESCRIPTION
**Pull request by snwhite**

### CODE REVIEW

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -238,6 +238,7 @@ CGCON-GWFPGX-99701,A00268.1,Sensor,0,Global Wire-Following Profiler Controller,W
 CGCON-MCPM03-00001,OL000181,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,1,,,,"CE01ISSM-0000(1,2)-CPM3",Oregon State University
 CGCON-MCPM03-00002,OL000485,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,0002,,,,CP01CNSM-00001-CPM3,Woods Hole Oceanographic Institution
 CGCON-MCPM03-00003,,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,3,,,,,
+CGCON-MCPM03-00004,OL000499,Sensor,0,Multi-Function Node Communications and Power Manager,Communications and Power Manager,WHOI,CPM,4,,,,,Oregon State University
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv:241]: looks correct

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -323,6 +324,7 @@ CGCON-NCPM02-00004,OL000380 OL000498 OL000531,Sensor,0,Near Surface Instrument F
 CGCON-NCPM02-00005,OL000567,Sensor,0,Near Surface Instrument Frame Communications and Power Manager,Communications and Power Manager,WHOI,CPM,0005,,,,CP03ISSM-00001-CPM2,Woods Hole Oceanographic Institution
 CGCON-NCPM02-00006,OL000555,Sensor,0,Near Surface Instrument Frame Communications and Power Manager,Communications and Power Manager,WHOI,CPM,0006,,,,CP04OSSM-00003-CPM2,Woods Hole Oceanographic Institution
 CGCON-NCPM02-00007,OL000394,Sensor,0,Near Surface Instrument Frame Communications and Power Manager,Communications and Power Manager,WHOI,CPM,3703-00274-0007,,,,CE04OSSM-00002-RIC21,Oregon State University
+CGCON-NCPM02-00008,,Sensor,0,Near Surface Instrument Frame Communications and Power Manager,,WHOI,CPM,8,,,,,Oregon State University
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv:327]: looks correct

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -473,6 +475,12 @@ OL000306,OL000306,Sensor,0,Near Surface Instrument Frame Data Concentrator Logge
...
 OL000312,OL000312,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,,,,,CE02SHSM-00002-DCL11,Oregon State University
 OL000313,OL000313,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,,,,,CE02SHSM-00002-DCL12,Oregon State University
+OL000379,OL000379,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,,,,,,Oregon State University
+OL000380,OL000380,Sensor,0,Near Surface Instrument Frame Communications and Power Manager,Communications and Power Manager,WHOI,CPM,,,,,,Oregon State University
+OL000381,OL000381,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,,,,,,Oregon State University
+OL000382,OL000382,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,,,,,,Oregon State University
+OL000383,OL000383,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,,,,,,Oregon State University
+OL000384,OL000384,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,,,,,,Oregon State University
```
Comment (rtravis-whoi) [inline bulk/eng_bulk_load-AssetRecord.csv:483]: the 'OL' additions look correct

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -2541,3 +2541,4 @@ PIRSN-OVRSRA-00001,,Sensor,0,Overview Sonar,OVRSR Series A,MARUM,R2 Sonic 2022,1
 PIRSN-QNTSRA-00001,,Sensor,0,Quantification Sonar,QNTSR Series A,MARUM,Kongsberg 1171-Series Scanning Sonar,1708062,,,,,
 PIRSN-SCPRAA-00001,,Sensor,0,Self-Calibrating Pressure Recorder,SCPR Series A,PI Zumberge and Sasagawa,,001/C,,,,,
 PIRSN-SCTAAA-00001,,Sensor,0,Self-Calibrating Triaxial Accelerometer,SCTA Series A,PI Wilcock,,1,,,,,
+R00007,R00007,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,,,,,,Oregon State University
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2544]: addition looks correct

**Review by rtravis-whoi — APPROVED** [review]
looks good!

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:241]: Looks good

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:327]: Looks good

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:483]: Looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 4 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:2544]: Looks good

File: calibration/METBKA/N00747__20141213.csv [hunk 5]
```diff
@@ -1,9 +0,0 @@
-serial,name,value,notes
-CP01CNSM-00002-METBK1,CC_depth_of_conductivity_and_temperature_measurements_m,0,Depth of sea temperature measurement TEMPSRF [m]
-CP01CNSM-00002-METBK1,CC_height_of_air_humidity_measurement_m,4,height of air humidity measurement RELHUMI [m]
-CP01CNSM-00002-METBK1,CC_height_of_air_temperature_measurement_m,4,Height of air temperature measurement TEMPAIR [m]
-CP01CNSM-00002-METBK1,CC_height_of_windspeed_sensor_above_sealevel_m,4,Height of windspeed measurement WINDAVG_L0 [m]
-CP01CNSM-00002-METBK1,CC_jcool,1,do coolskin calc
-CP01CNSM-00002-METBK1,CC_jwarm,1,do warmlayer calc
-CP01CNSM-00002-METBK1,CC_zinvpbl,600,"planetary boundary layer/inversion height: this is set to a default value of 600m in the code, as is used in the DPS."
-CP01CNSM-00002-METBK1,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/N00747__20141213.csv:9]: Deletion looks good

File: calibration/METBKA/N00748__20141213.csv [hunk 6]
```diff
@@ -1,9 +0,0 @@
-serial,name,value,notes
-CP01CNSM-00002-METBK2,CC_depth_of_conductivity_and_temperature_measurements_m,0,Depth of sea temperature measurement TEMPSRF [m]
-CP01CNSM-00002-METBK2,CC_height_of_air_humidity_measurement_m,4,height of air humidity measurement RELHUMI [m]
-CP01CNSM-00002-METBK2,CC_height_of_air_temperature_measurement_m,4,Height of air temperature measurement TEMPAIR [m]
-CP01CNSM-00002-METBK2,CC_height_of_windspeed_sensor_above_sealevel_m,4,Height of windspeed measurement WINDAVG_L0 [m]
-CP01CNSM-00002-METBK2,CC_jcool,1,do coolskin calc
-CP01CNSM-00002-METBK2,CC_jwarm,1,do warmlayer calc
-CP01CNSM-00002-METBK2,CC_zinvpbl,600,"planetary boundary layer/inversion height: this is set to a default value of 600m in the code, as is used in the DPS."
-CP01CNSM-00002-METBK2,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/N00748__20141213.csv:9]: Deletion looks good

File: calibration/METBKA/N00749__20150507.csv [hunk 7]
```diff
@@ -1,9 +0,0 @@
-serial,name,value,notes
-CP01CNSM-00003-METBK1,CC_depth_of_conductivity_and_temperature_measurements_m,0,Depth of sea temperature measurement TEMPSRF [m]
-CP01CNSM-00003-METBK1,CC_height_of_air_humidity_measurement_m,4,height of air humidity measurement RELHUMI [m]
-CP01CNSM-00003-METBK1,CC_height_of_air_temperature_measurement_m,4,Height of air temperature measurement TEMPAIR [m]
-CP01CNSM-00003-METBK1,CC_height_of_windspeed_sensor_above_sealevel_m,4,Height of windspeed measurement WINDAVG_L0 [m]
-CP01CNSM-00003-METBK1,CC_jcool,1,do coolskin calc
-CP01CNSM-00003-METBK1,CC_jwarm,1,do warmlayer calc
-CP01CNSM-00003-METBK1,CC_zinvpbl,600,"planetary boundary layer/inversion height: this is set to a default value of 600m in the code, as is used in the DPS."
-CP01CNSM-00003-METBK1,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/N00749__20150507.csv:9]: Deletion looks good

File: calibration/METBKA/N00750__20150507.csv [hunk 8]
```diff
@@ -1,9 +0,0 @@
-serial,name,value,notes
-CP01CNSM-00003-METBK2,CC_depth_of_conductivity_and_temperature_measurements_m,0,Depth of sea temperature measurement TEMPSRF [m]
-CP01CNSM-00003-METBK2,CC_height_of_air_humidity_measurement_m,4,height of air humidity measurement RELHUMI [m]
-CP01CNSM-00003-METBK2,CC_height_of_air_temperature_measurement_m,4,Height of air temperature measurement TEMPAIR [m]
-CP01CNSM-00003-METBK2,CC_height_of_windspeed_sensor_above_sealevel_m,4,Height of windspeed measurement WINDAVG_L0 [m]
-CP01CNSM-00003-METBK2,CC_jcool,1,do coolskin calc
-CP01CNSM-00003-METBK2,CC_jwarm,1,do warmlayer calc
-CP01CNSM-00003-METBK2,CC_zinvpbl,600,"planetary boundary layer/inversion height: this is set to a default value of 600m in the code, as is used in the DPS."
-CP01CNSM-00003-METBK2,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/N00750__20150507.csv:9]: Deletion looks good

File: calibration/METBKA/N00754__20150509.csv [hunk 9]
```diff
@@ -1,9 +0,0 @@
-serial,name,value,notes
-CP04OSSM-00002-METBK,CC_depth_of_conductivity_and_temperature_measurements_m,1.0668,
-CP04OSSM-00002-METBK,CC_height_of_air_humidity_measurement_m,4.2926,
-CP04OSSM-00002-METBK,CC_height_of_air_temperature_measurement_m,4.2926,
-CP04OSSM-00002-METBK,CC_height_of_windspeed_sensor_above_sealevel_m,4.7498,
-CP04OSSM-00002-METBK,CC_jcool,1,Constant
-CP04OSSM-00002-METBK,CC_jwarm,1,Constant
-CP04OSSM-00002-METBK,CC_zinvpbl,600,Constant
-CP04OSSM-00002-METBK,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/N00754__20150509.csv:9]: Deletion looks good

File: calibration/METBKA/N00756__20150509.csv [hunk 10]
```diff
@@ -1,9 +0,0 @@
-serial,name,value,notes
-CP03ISSM-00002-METBK,CC_depth_of_conductivity_and_temperature_measurements_m,1.0668,S/N to be determined after recovery
-CP03ISSM-00002-METBK,CC_height_of_air_humidity_measurement_m,4.2926,S/N to be determined after recovery
-CP03ISSM-00002-METBK,CC_height_of_air_temperature_measurement_m,4.2926,S/N to be determined after recovery
-CP03ISSM-00002-METBK,CC_height_of_windspeed_sensor_above_sealevel_m,4.7498,S/N to be determined after recovery
-CP03ISSM-00002-METBK,CC_jcool,1,Constant
-CP03ISSM-00002-METBK,CC_jwarm,1,Constant
-CP03ISSM-00002-METBK,CC_zinvpbl,600,Constant
-CP03ISSM-00002-METBK,CC_use_velpt,1,
```
Comment (cdobs) [inline calibration/METBKA/N00756__20150509.csv:9]: Deletion looks good

**Review by cdobs — APPROVED** [review]
All looks good to me

**Pull request merged**